manager/src/main/java/com/redhat/service/smartevents/manager/ProcessorService.java

```diff
@@ -15,6 +15,8 @@ public interface ProcessorService {
 
     Processor createProcessor(String bridgeId, String customerId, String owner, ProcessorRequest processorRequest);
 
+    Processor createErrorHandlingProcessor(String bridgeId, String customerId, String owner, ProcessorRequest processorRequest);
```

Comment:
Creation of an Error Handling `Processor` is a little different. It needs to set the `isErrorHandler` flag on the `ProcessorDefinition`. Rather than add an additional parameter to the existing method I added a new method to make its action more explicit.